core/fafshare.gms

```diff
@@ -51,26 +51,26 @@ set
 		43	"Mixed�freight" /,
 
     yr 		Years in WiNDC Database,
-    fyr		Years in FAF data (1997-2021),
+*    fyr(yr)		Years in FAF data (1997-2022),
```

Comment:
I've set the domain of faf_units to explicitly be yr in the data side. I've kept this to make minimal changes to the code, fyr is used below.